### PR TITLE
Feature/audioservice list

### PR DESCRIPTION
**Pull request by forslund**
## Description
The AudioService object now has the new method available_backends() returning a dict of all available audio backends as keys containing 'supported_uris' listing supported uri types (file:// http:// etc.), 'remote' (bool) True if remote), 'default' (bool) true if the default audio backend.

## How to test
Ensure the AudioService.available_backends() returns a valid response

## Contributor license agreement signed?
CLA [ Yes ]

### CODE REVIEW

File: mycroft/audio/audioservice.py [hunk 1]
```diff
@@ -496,4 +509,3 @@ def shutdown(self):
         self.bus.remove('recognizer_loop:audio_output_end',
                         self._restore_volume)
         self.bus.remove('recognizer_loop:record_end', self._restore_volume)
-        self.bus.remove('mycroft.stop', self._stop)
```
Comment (MatthewScholefield) [inline mycroft/audio/audioservice.py]: Any reason for this change?

Comment (forslund) [reply mycroft/audio/audioservice.py]: Yes, it's removed but it's never added so it's sort of redundant (I put it in a separate commit to differentiate it from the other change)

**Comment by MatthewScholefield** [comment]
Where would I see the message reply? I'm doing `tail -f /var/log/mycroft/*.log` and all I see when I do `python3 -m mycroft.messagebussend mycroft.audio.service.list_backends` is a single message about chromecasts:
```
10:40:45.562 - chromecast__init__:supported_uris:67 - INFO - Chromecasts found: Chromecast(...)
```

**Comment by forslund** [comment]
I do think the logging of the messagebus is broken. @penrods was looking into that.

**Comment by forslund** [comment]
Try adding this to an intent handler:
```python
        from mycroft.skills.audioservice import AudioService
        print(AudioService(self.bus).available_backends())
```

**Pull request merged**